### PR TITLE
ENH: Add local orientation and uniform coloring to `scil_visualize_bundles`

### PR DESCRIPTION
**Pull request by AntoineTheb**
As title says. Default coloring (by fury-gl) remains to compute a single color for each streamlines based on its endpoints

If you have the rbx atlas downloaded, you can test with:

Local coloring:
`$ ./scripts/scil_visualize_bundles.py atlas/**/* --local_coloring --width=0.35 --subsample 5`
image

Uniform coloring:
`$ ./scripts/scil_visualize_bundles.py atlas/**/* --width=0.35 --subsample 5 --uniform_coloring 50 100 85`
image

Default coloring:
`$ ./scripts/scil_visualize_bundles.py atlas/**/* --width=0.35 --subsample 5`
image

@arnaudbore @frheault @GuillaumeTh

### CODE REVIEW

**Comment by frheault** [comment]
@arnaudbore I approved

File: scripts/scil_visualize_bundles.py [hunk 1]
```diff
@@ -47,6 +47,12 @@ def _build_arg_parser():
     coloring_group = p.add_mutually_exclusive_group()
```
[the remark this answers is not in the document]
Comment (arnaudbore) [reply scripts/scil_visualize_bundles.py:47]: Coloring method - something like this.

File: scripts/scil_visualize_bundles.py [hunk 2]
```diff
@@ -47,6 +47,12 @@ def _build_arg_parser():
...
     coloring_group.add_argument('--random_coloring', metavar='SEED', type=int,
                                 help='Assign a random color to bundles.')
+    coloring_group.add_argument('--uniform_coloring', metavar='R G B',
+                                nargs='+',
```
Comment (arnaudbore) [inline scripts/scil_visualize_bundles.py:51]: should it be 3 here instead of '+'

Comment (AntoineTheb) [reply scripts/scil_visualize_bundles.py]: Resolved in d93a3de

File: scripts/scil_visualize_bundles.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (AntoineTheb) [reply scripts/scil_visualize_bundles.py]: Resolved in 77d8704

**Review by arnaudbore — APPROVED** [review]

**Pull request merged**